multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y

```

Comment:
I think this needs to be a multiply sign, rather than an addition. This should hopefully fix the error!

```suggestion
    return x * y
```